vnext/Microsoft.ReactNative/Fabric/Composition/CompositionViewComponentView.cpp

```diff
@@ -1091,6 +1091,9 @@ void CompositionBaseComponentView::updateAccessibilityProps(
       UIA_IsEnabledPropertyId,
       !oldViewProps.accessibilityState.disabled,
       !newViewProps.accessibilityState.disabled);
+
+  winrt::Microsoft::ReactNative::implementation::UpdateUiaProperty(
```

Comment:
I genuinely don't know if this is the right call. Generally speaking I don't think ATs expect control type to change and aren't likely to listen to prop change notifications for ControlType.  I feel like I saw MSDN guidance on this, but all I can find in this moment is [a bit more ambiguous](https://learn.microsoft.com/en-us/windows/win32/winauto/uiauto-propertiesoverview). I don't think there's much harm in this and may be the right call given how react-native can make this property come in late due to a rerender or something, but figured I should at least speculate openly on it :)

Reply:
Makes sense! I'll do a bit of reading and check what behavior we set on Paper!

Reply:
Looks like on Paper we did check for and make updates to accessibilityRole in the UpdateProperty() functions. Maybe for now we leave as is to match the Paper behavior? But definitely a good callout. It's possible we were doing the wrong thing before and should keep that on our radar.

Reply:
Sounds good to me